wasm-engines/docker/wasm3.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ewasm/llvm-10:1.0
+
+RUN git clone https://github.com/ewasm-benchmarking/wasm3.git --single-branch --branch benchmark && \
+    cd wasm3 && mkdir build && cd build && cmake -DWASI=none -DCMAKE_BUILD_TYPE=Release .. && make -j4
```

Comment:
```suggestion
    cd wasm3 && mkdir build && cd build && cmake -DBUILD_WASI=none -DCMAKE_BUILD_TYPE=Release .. && make -j4
```

Reply:
As discussed, we need to rebase the wasm3 benchmarking branch on top of a newer version of wasm3 to include this.  Let's leave this, merge this PR and open an issue.